cineast-core/src/main/java/org/vitrivr/cineast/core/features/abstracts/AbstractSegmentExporter.java

```diff
@@ -0,0 +1,124 @@
+package org.vitrivr.cineast.core.features.abstracts;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.function.Supplier;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.vitrivr.cineast.core.data.segments.SegmentContainer;
+import org.vitrivr.cineast.core.db.PersistencyWriterSupplier;
+import org.vitrivr.cineast.core.db.setup.EntityCreator;
+import org.vitrivr.cineast.core.features.extractor.Extractor;
+import org.vitrivr.cineast.core.util.LogHelper;
+
+abstract public class AbstractSegmentExporter implements Extractor {
+
+  protected static final Logger LOGGER = LogManager.getLogger();
+
+  private static final String PROPERTY_NAME_DESTINATION = "destination";
+
+  /**
+   * Destination path for the audio-segment.
+   */
+  private final Path destination;
+
+  protected String fileExtension;
+
+  protected String dataUrlPrefix;
+
+  /**
+   * Default constructor
+   */
+  public AbstractSegmentExporter() {
+    this(new HashMap<>());
+  }
+
+  /**
+   * Default constructor. The AudioSegmentExport can be configured via named properties in the provided HashMap.
+   * <p>
+   * Supported parameters:
+   *
+   * <ol>
+   *      <li>destination: Path where files should be stored.</li>
```

Comment:
I would highlight in the documentation that its literally `destination`

Reply:
Can you please clarify this?

Reply:
You have `destination` as the property key. I would expect that this is marked as such (e.g. with `@code` in line 57)